src/txn/txn_timestamp.c

```diff
@@ -1151,6 +1219,124 @@ __wt_txn_clear_commit_timestamp(WT_SESSION_IMPL *session)
 	WT_PUBLISH(txn->flags, flags);
 }
 
+/*
```

Comment:
This section is more or less copied verbatim from the `commit` versions of these functions.